packages/frontend-app/src/app/components/CookieBanner.tsx

```diff
@@ -1,13 +1,25 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import classNames from "classnames";
 
 import { Button } from "./Button";
 import useCookieConsent from "../hooks/useCookieConsent";
 
 export const CookieBanner = () => {
   const [value, consentToCookie] = useCookieConsent();
+  const [isVisible, setIsVisible] = useState(false);
 
-  return !value ? (
+  useEffect(() => {
+    if (!value) {
+      setIsVisible(false);
+    }
+  }, [value]);
```

Comment:
This is not necessary. Having this causes a second render after mount which is not necessary.

Reply:
ok, thanks